nodejs/tslint.json

```diff
@@ -57,7 +56,6 @@
         "no-internal-module": true,
         "no-parameter-properties": false,
         "no-require-imports": false,
-        "no-shadowed-variable": true,
```

Comment:
we've disabled this in other repos already.  it's really obnoxious with outputs/apply where it is actually super nice to just be able to reuse the same name for the output and the unwrapped value.